### PR TITLE
Documentation pass 10

### PR DESCRIPTION
**Pull request by alganthe**
Updated the documentation for ACE 3.2.0

### CODE REVIEW

**Comment by alganthe** [comment]
All typos are now fixed.

File: documentation/feature/finger.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+## 1. Overview
+Allows players to point in a direction with their fingers, when they do so people around (4m by default) can see a big circle in the pointed direction.
+
+
+## 2. Usage
+
+### 2.1 How to point things
+- Press <kbd>⇧&nbsp;Shift</kbd>+<kbd>`</kbd>.
```
Comment (bux578) [inline documentation/feature/finger.md]: Maybe indicate on which keyboard layout

File: documentation/feature/parachute.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ Removes the altitude and descend speed UI elements when free-falling and parachu
 ### 1.2 Non-steerable parachute
 Adds a non-steerable parachute variant for jet pilots.
 
-### 1.3 Landing animation
-Smoothens the parachute landing animation.
+### 1.3 Parachute cutting and reserve parachutes
+You are now able to cut parachutes and deploy a reserve one.
 
-### 1.4 Reserve parachute
```
Comment (bux578) [inline documentation/feature/parachute.md]: There's still a reserve parachute once you've cut the primary one

File: documentation/feature/slideshow.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+group: feature
+parent: wiki
+---
+
+## 1. Overview
+This adds the ability to have images shown on some objects and have other objects being used as remotes.
+Please note that only objects with hiddenSelection 0 can be used to render images (whiteboard, TV, PC Screen being the most notable examples).
+
+## 2. Usage
+Note that this sections is for users, for mission makers refer to [the entry in mission-tools](http://ace3mod.com/wiki/missionmaker/mission-tools.html)
```
Comment (bux578) [inline documentation/feature/slideshow.md]: it should be enough to use a relative url

File: documentation/feature/parachute.md (hunk 2 above)
Comment (alganthe) [inline documentation/feature/parachute.md]: you commented on the old diff :D the new one indicates that you can deploy the reserve parachute after cutting the primary.

**Comment by alganthe** [comment]
Typos fixed.

**Pull request merged**